### PR TITLE
NDEV-496 Create method neon_getEvmParams

### PR DESCRIPTION
**Pull request by ivandzen**

### CODE REVIEW

File: proxy/neon_rpc_api_model/neon_rpc_api_model.py [hunk 1]
```diff
@@ -697,3 +697,17 @@ def neon_emulate(self, raw_signed_trx):
         eth_trx = EthTrx.fromString(bytearray.fromhex(raw_signed_trx))
         emulation_result = call_trx_emulated(eth_trx)
         return emulation_result
+
+    def neon_getEvmParams(self, param_name):
```
Comment (afalaleev) [inline proxy/neon_rpc_api_model/neon_rpc_api_model.py]: Why the name of the method is `neon_getEvParamS`, but it returns a param by name?

Comment (afalaleev) [reply proxy/neon_rpc_api_model/neon_rpc_api_model.py]: I suggested that the method should return the entire parameter map, and the parameter should be easy to get by name on the client side.

Comment (afalaleev) [reply proxy/neon_rpc_api_model/neon_rpc_api_model.py]: Please, define the return type: `def neon_getEvmParams(self) -> Dict[str, Any]:`

**Review by afalaleev — CHANGES_REQUESTED** [review]

File: proxy/neon_rpc_api_model/neon_rpc_api_model.py (hunk 1 above)
Comment (ivandzen) [reply proxy/neon_rpc_api_model/neon_rpc_api_model.py]: Fixed

File: proxy/testing/test_get_evm_params.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
+import unittest
+from ..common_neon.web3 import NeonWeb3 as Web3
+from ..common_neon.elf_params import ElfParams
```
Comment (rozhkovdmitrii) [inline proxy/testing/test_get_evm_params.py]: not used

File: proxy/testing/test_get_evm_params.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+proxy_url = os.environ.get('PROXY_URL', 'http://127.0.0.1:9090/solana')
+proxy = Web3(Web3.HTTPProvider(proxy_url))
+
+class TestGetEvmParam(unittest.TestCase):
+    def test_all_cases(self):
+        print(f'Neon-EVM Params: {proxy.neon.getEvmParams()}')
```
Comment (rozhkovdmitrii) [inline proxy/testing/test_get_evm_params.py:10]: Don't forget please to update documentation

File: proxy/neon_rpc_api_model/neon_rpc_api_model.py [hunk 4]
```diff
@@ -697,3 +697,10 @@ def neon_emulate(self, raw_signed_trx):
         eth_trx = EthTrx.fromString(bytearray.fromhex(raw_signed_trx))
         emulation_result = call_trx_emulated(eth_trx)
         return emulation_result
+
+    def neon_getEvmParams(self)-> Dict[str, Any]:
+        """
```
Comment (rozhkovdmitrii) [inline proxy/neon_rpc_api_model/neon_rpc_api_model.py]: If takes single line let it be a single line

File: proxy/common_neon/web3.py [hunk 5]
```diff
@@ -20,6 +20,13 @@ def emulate_munger(
         mungers=[emulate_munger],
     )
 
+    neon_getEvmParams = RPCEndpoint("neon_getEvmParams")
+
+    getEvmParams = Method(
+        neon_getEvmParams,
+        mungers=[],
+    )
+
```
Comment (rozhkovdmitrii) [inline proxy/common_neon/web3.py:29]: two blank lines

**Review by afalaleev — DISMISSED** [review]

File: proxy/common_neon/web3.py (hunk 5 above)
Comment (ivandzen) [reply proxy/common_neon/web3.py:29]: done

File: proxy/neon_rpc_api_model/neon_rpc_api_model.py (hunk 4 above)
Comment (ivandzen) [reply proxy/neon_rpc_api_model/neon_rpc_api_model.py]: done

File: proxy/testing/test_get_evm_params.py (hunk 3 above)
Comment (ivandzen) [reply proxy/testing/test_get_evm_params.py:10]: Will do

File: proxy/testing/test_get_evm_params.py (hunk 2 above)
Comment (ivandzen) [reply proxy/testing/test_get_evm_params.py]: removed

**Review by rozhkovdmitrii — APPROVED** [review]

**Review by rozhkovdmitrii — DISMISSED** [review]

**Review by rozhkovdmitrii — APPROVED** [review]

**Pull request merged**